README.rst

```diff
@@ -95,6 +99,18 @@ To run javascript linter watcher:
 
 ``gulp lint:watch``
 
+To run javascript code style analysis:
+
+``gulp jscs``
```

Comment:
We have the command to do it together: gulp ci
But at this moment, it's better 4 me to do it separately, because at first, I will fix styling errors and then, I will fix/replace the code logic.